doc/how_to_guides/moveit_launch_files/moveit_launch_files_tutorial.rst

```diff
@@ -213,3 +248,23 @@ Finally, we can tell our launch file to actually launch everything we described
                 hand_controller_spawner,
             ]
         )
+
+Launching a custom .cpp file
+----------------------------
+
+While not part of the Getting Started tutorial, another common node to launch is one that executes a custom .cpp file:
```

Comment:
A custom `.cpp` file doesn't sound technically precise -- I think what you may want to say is "a custom executable.

Also, this in isolation may not be entirely helpful if you also don't show a `CMakeLists.txt` snippet showing how to actually compile this executable.

But generally, there are other tutorials like https://moveit.picknik.ai/main/doc/tutorials/your_first_project/your_first_project.html which already cover this in full detail.